docs/Changelog.md

```diff
@@ -8855,6 +8855,42 @@ This version of the operator has been available since version 9 of the default O
 <dd>Constrains output to boolean tensor.</dd>
 </dl>
 
+### <a name="GreaterOrEqual-9"></a>**GreaterOrEqual-9**</a>
+
+  Returns the tensor resulted from performing the `greater or equal` logical operation
```

Comment:
I suggest slight rephrasing: "Returns tensor resulting from..."

Reply:
ok sure, this will update the docs of all logical ops, and I think it is fine =w=